### DISCUSSION TITLE
Cannot remap Meta+L on Windows

### DISCUSSION THREAD

**Discussion by ghost** [houmain/keymapper#199] [Q&A] [answered]
Not sure what's so special about this key combo but all other Meta-type shortcuts can be remapped besides this one. (Meta L) or Meta{L} always locks regardless of mapping.

**Comment by ghost** [comment, marked as answer]
Apprently it's windows thing, you can't remap it in powertoys either.

**Comment by kbilsted** [comment]
Yes there are certain key sequences that is hard-coded at the OS level and cannot be remapped. It should be in the documentation

**Reply by kbilsted** [reply]
i made a PR

**Reply by Zireael07** [reply]
Still not documented and I just ran into this on my work laptop :/

**Comment by ristomatti** [comment]
A firmware level remappable keyboard is blessing in these kind of situations. But even though I have one, I prefer using Keymapper for everything else due to its flexibility. 🙂
